app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragmentHelper.kt

```diff
@@ -93,6 +94,26 @@ fun ResourcesFragment.setupAndShowSelectedResourcesMenu(menuBtn: View) {
                 .show(parentFragmentManager, null)
             popup.popupWindow.dismiss()
         }
+        with(btnIncreaseScore) {
+            isVisible = presenter.allowScoring()
+            setOnClickListener {
+                presenter.onIncreaseScoreClicked()
+                popup.popupWindow.dismiss()
+            }
+        }
+        with(btnDecreaseScore) {
+            isVisible = presenter.allowScoring()
+            setOnClickListener {
+                presenter.onDecreaseScoreClicked()
+                popup.popupWindow.dismiss()
+            }
+        }
```

Comment:
I think popup can be left after changing score

Reply:
Oh! yeah, sounds nice